components/core/gunicorn.py.ini

```diff
@@ -0,0 +1,14 @@
+"""gunicorn WSGI server configuration."""
+from multiprocessing import cpu_count
+from os import environ
+
+
+def max_workers():    
+    return cpu_count()
```

Comment:
2 * cpu_count + 1 is the optimized value I think
https://medium.com/building-the-system/gunicorn-3-means-of-concurrency-efbb547674b7